integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataConnector.java

```diff
@@ -31,13 +31,13 @@
   private static final Logger log = Logger.get(CarbondataConnector.class);
 
   private final LifeCycleManager lifeCycleManager;
-  private final CarbondataMetadata metadata;
+  private final ConnectorMetadata metadata;
```

Comment:
what are the difference of using CarbonDataMetadata or ConnectorMetadata?

Reply:
CarbonDataMetadata extends ConnectorMetaData, we should always code to Interface hence have changed this, also this way in concurrency  we are using the standard Presto protocol and providing a ClassLoaderSafeConnectorMetadata

Reply:
ok